client/src/components/FormEducation.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import FormSingleInputArray from "./FormSingleInputArray";
+
+export default function FormEducation() {
+  const [number, setNumber] = useState(0);
+
+  //Creating array so I can make a map for profile inputs
+  let tempArray = [];
+  for (let i = 0; i < number; i++) {
+    tempArray.push(i);
+  }
+
+  const handleChange = () => {
+    let inputValue = document.querySelector(".numberOfEducations").value;
+    setNumber(inputValue);
+  };
+
+  let educations = tempArray.map((work, i) => {
```

Comment:
I love how you utilized React and mapping to make a dynamic number of inputs for your forms.